benchmarks/bench_gdn_prefill.py

```
@@ -100,7 +100,18 @@ def bench_fi(endpoints, h_qk, h_v, d, warmup, iters):
+        k,
+        v,
+        g,
+        beta,
+        scale=None,
+        initial_state=h0,
+        output_final_state=True,
+        cu_seqlens=cu_seqlens,
+        use_qk_l2norm_in_kernel=False,
+        output=None,
+        output_state=state_out,
     )
```

Ruff (0.15.11)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `fn` as a `def`

(E731)